src/hooks/useReadIdentityRegistryInvestorCountry.tsx

```diff
@@ -0,0 +1,23 @@
+import { EvmAddress } from "@/types/types";
+import { QueryKeys } from "@/hooks/types";
+import { useQuery } from "@tanstack/react-query";
+import { readIdentityRegistryInvestorCountry } from "@/services/contracts/wagmiGenActions";
+
+export function useReadIdentityRegistryInvestorCountry(
```

Comment:
we already have this hook in `main` branch

src/hooks/eip3643/useReadIdentityRegistryInvestorCountry.ts